src/config.py

```diff
@@ -22,7 +22,7 @@ class Config:
     MONGOD_CONF_DIR = "/etc/mongod"
     MONGODB_LOG_FILENAME = "mongodb.log"
 
-    CHARM_INTERNAL_VERSION_FILE = "charm_internal_version"
+    CHARM_INTERNAL_VERSION_FILE = "charm_version"
```

Comment:
:purple_heart: 

Hopefully we soon can publish a new version of Data Helpers which removes the difference between charm internal version and charm version since it's not meaningful anymore.